api/classes/publisher.js

```diff
@@ -310,6 +318,23 @@ class BasePublisher {
     return publishedFilesQueryBuilder
     .getAllModifiedFiles(this.publishedProject.id)
     .then(function(publishedFiles) {
```

Comment:
same thing here:
`.then(publishedFiles => {`

and inside you can use `this.update...`